.github/workflows/submitProduction.yml

```diff
@@ -63,51 +51,59 @@ jobs:
-      - name: Create Pull Request
-        uses: peter-evans/create-pull-request@7380612b49221684fefa025244f2ef4008ae50ad #v3.10.1
-        with:
-          title: Bump versions to ${{ github.event.inputs.version }}
-      - name: Create GitHub Release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-        with:
-          tag_name: ${{ github.event.inputs.version }}
-          release_name: ${{ github.event.inputs.version }}
-          body: ${{ github.event.inputs.version }}
-          draft: false
-          prerelease: false
-      - name: Upload Release Asset
-        id: upload-release-asset
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-        with:
-          upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: ./extension/build/build.zip
-          asset_name: build-${{ github.event.inputs.version }}.zip
-          asset_content_type: application/zip
-      - name: Submit extension to Firefox
-        uses: kewisch/action-web-ext@a28b4ba2f491cc36c67a714a716ca54218440056 #v1
+      # - name: Commit files
+      #   run: |
+      #     git config --local user.email "action@github.com"
+      #     git config --local user.name "GitHub Action"
+      #     git add -A
+      #     git commit -m "docs(): bumping release to ${{ github.event.inputs.version }}"
+      #     git tag ${{ github.event.inputs.version }}
+      # - name: Create Pull Request
+      #   uses: peter-evans/create-pull-request@7380612b49221684fefa025244f2ef4008ae50ad #v3.10.1
+      #   with:
+      #     title: Bump versions to ${{ github.event.inputs.version }}
+      # - name: Create GitHub Release
+      #   id: create_release
+      #   uses: actions/create-release@v1
+      #   env:
+      #     GITHUB_TOKEN: ${{ github.token }}
+      #   with:
+      #     tag_name: ${{ github.event.inputs.version }}
+      #     release_name: ${{ github.event.inputs.version }}
+      #     body: ${{ github.event.inputs.version }}
+      #     draft: false
+      #     prerelease: false
+      # - name: Upload Release Asset
+      #   id: upload-release-asset
+      #   uses: actions/upload-release-asset@v1
+      #   env:
+      #     GITHUB_TOKEN: ${{ github.token }}
+      #   with:
+      #     upload_url: ${{ steps.create_release.outputs.upload_url }}
+      #     asset_path: ./extension/build/build.zip
+      #     asset_name: build-${{ github.event.inputs.version }}.zip
+      #     asset_content_type: application/zip
+      # - name: Submit extension to Firefox
+      #   uses: kewisch/action-web-ext@a28b4ba2f491cc36c67a714a716ca54218440056 #v1
+      #   with:
+      #     cmd: sign
+      #     source: ./extension/build/build.zip
+      #     channel: listed
+      #     apiKey: ${{ secrets.AMO_SIGN_KEY }}
+      #     apiSecret: ${{ secrets.AMO_SIGN_SECRET }}
+      #     timeout: 900000
```

Comment:
commenting out all the Firefox and GH release stuff as we already have this for for 5.19.1